### PR TITLE
The Per Disk Read/Write Per Sec Metric was removed

### PR DESCRIPTION
**Pull request by agrare**
Replace with Disk Read Bytes and Disk Write Bytes, sum and convert to kb/sec

Also fix Network In Total and Network Out Total not converting to a rate.

Fixes https://github.com/ManageIQ/manageiq-providers-azure/issues/434

### CODE REVIEW

**Comment by agrare** [comment]
We do basically the same conversion in the Amazon provider: https://github.com/ManageIQ/manageiq-providers-amazon/blob/master/app/models/manageiq/providers/amazon/cloud_manager/metrics_capture.rb#L12-L21

Note this is going to fail specs, need to update the VCRs

File: spec/models/manageiq/providers/azure/cloud_manager/metrics_capture_spec.rb [hunk 1]
```diff
@@ -138,40 +138,87 @@
...
+          },
+          "2016-07-23T07:22:00Z" => {
+            "disk_usage_rate_average" => expected_usage_kb_per_sec
+          }
+        }
+      }
+      expect(metrics_by_id_and_ts).to eq(expected_metrics)
+    end
+
+    it "parses and aggregates network inout/output" do
```
Comment (Fryguy) [inline spec/models/manageiq/providers/azure/cloud_manager/metrics_capture_spec.rb:184]:
```suggestion
    it "parses and aggregates network input/output" do
```

**Review by Fryguy — APPROVED** [review]

File: spec/models/manageiq/providers/azure/cloud_manager/metrics_capture_spec.rb (hunk 1 above)
Comment (agrare) [reply spec/models/manageiq/providers/azure/cloud_manager/metrics_capture_spec.rb]: :+1: the keys are like...right next to each other

File: spec/models/manageiq/providers/azure/cloud_manager/metrics_capture_spec.rb [hunk 2]
```diff
@@ -138,40 +138,87 @@
...
-        build_metric_data(982_252_000, "2016-07-23T07:22:00.5580968Z")
+        build_metric_data(disk_read_write_bytes, "2016-07-23T07:20:00.5580968Z"),
+        build_metric_data(disk_read_write_bytes, "2016-07-23T07:21:00.5580968Z"),
+        build_metric_data(disk_read_write_bytes, "2016-07-23T07:22:00.5580968Z")
       ]
       stage_metrics(metric_data, counters)
 
       _, metrics_by_id_and_ts = vm.perf_collect_metrics('interval_name')
 
+      expected_usage_kb_per_sec = (2 * disk_read_write_bytes) / 1.kilobyte / 1.minute
```
Comment (agrare) [inline spec/models/manageiq/providers/azure/cloud_manager/metrics_capture_spec.rb:154]: I thought this made it more obvious what we are checking for.  It wasn't immediately clear to me that `1_918_460` was `2 * 982_252_000 / 1024.`

**Review by djberg96 — APPROVED** [review]

**Comment by djberg96** [comment]
You'll want to backport this since all versions will be affected.

**Comment by simaishi** [comment]
Kasparov backport details:

```
$ git log -1
commit 35affd067806fb9f3bd35d5788e9590c4e57c12c
Author: Jason Frey <fryguy9@gmail.com>
Date:   Tue Feb 16 15:15:32 2021 -0500

    Merge pull request #436 from agrare/fix_missing_metric_per_disk_read_write_bytes_per_sec

    The Per Disk Read/Write Per Sec Metric was removed

    (cherry picked from commit b28630d9a123cb4f9dfc0297595cb0045f238e7c)
```

**Comment by simaishi** [comment]
Jansa backport details:

```
$ git log -1
commit fa6b2d2e970c842a4a4af1b28d2a382f1dfb6606
Author: Jason Frey <fryguy9@gmail.com>
Date:   Tue Feb 16 15:15:32 2021 -0500

    Merge pull request #436 from agrare/fix_missing_metric_per_disk_read_write_bytes_per_sec

    The Per Disk Read/Write Per Sec Metric was removed

    (cherry picked from commit b28630d9a123cb4f9dfc0297595cb0045f238e7c)
```

**Pull request merged**